src/archiefvernietigingscomponent/js/components/review-list/zaak-detail-modal.js

```diff
@@ -20,109 +19,46 @@ const ZaakDetailModal = ({ modalIsOpen, setIsOpen, zaak, index, comment, setComm
-                                    <h2 className="section-title section-title--highlight">Resultaat</h2>
-                                    {!resultaat
-                                        ? <span>Deze zaak heeft geen resultaat</span>
-                                        : <span title={ resultaat.toelichting }>{ resultaat.resultaattype.omschrijving }</span>
-                                    }
-                                </section>
-                            </div>
-                            <section className="content-panel">
-                                <h2 className="section-title section-title--highlight">Documenten</h2>
-                                {!documenten
-                                    ? <span>Deze zaak heeft geen gerelateerde documenten</span>
-                                    : <ul>
-                                        { documenten.map(
-                                            (document) => <li key={document.url}>{ document.identificatie }</li>
-                                        )}
-                                    </ul>
-                                }
-                            </section>
-
-                            <div className="zaak-detail__comment">
-                                <label htmlFor="id_comment">Opmerkingen:</label>
-                                <TextInput
-                                    id="id_comment"
-                                    name="comment"
-                                    initial={comment}
-                                    onChange={(event) => setComment(event.target.value)}
-                                />
-                            </div>
-
-                            <div className="modal__buttons">
-                                <button
-                                    type="button"
-                                    className="btn"
-                                    onClick={(e) => suggestClose("change_and_remove")}
-                                >Aanpassen
-                                </button>
-                                <button
-                                    type="button"
-                                    className="btn"
-                                    onClick={(e) => suggestClose("remove")}
-                                >Uitzonderen
-                                </button>
-                                <button
-                                    type="button"
-                                    className="btn"
-                                    onClick={(e) => suggestClose("")}
-                                >Akkoord
-                                </button>
-                            </div>
-
-                        </>
-                }
+                    <div className="zaak-detail__link">
+                        Link naar de zaak in de zaakafhandelcomponent (ZAC): <a href={zaak.zac_link}>{zaak.identificatie}</a>
```

Comment:
More visible indication (no need to write out what the link does)

```suggestion
                        {% if zaak.zac_link %}
                            Zaak: <a title="Open de geconfigureerde externe applicatie voor het bekijken van de zaak." target="_blank" href={zaak.zac_link}>{zaak.identificatie} <span class="material-icons-outlined">launch</span></a>
                        {% endif %}
```